README.md

```diff
@@ -108,11 +108,21 @@ You can also add secrets with `dotnet user-secrets`, just pipe the JSON you want
 cat secrets.json | dotnet user-secrets set
 ```
 
+### AWS Credentials
+
+In Rider:
+- Install the "AWS Toolkit" plugin
+- Open the credentials file in a text editor (this is usually `C:\\Users\<your username>\.aws\credentials`, 
+  or find the AWS tab in the bottom right of Rider, click "All local credentials" > "Edit AWS credential file(s)")
+- Log into AWS on a browser, and obtain the AWS profile to paste into the file (this varies between accounts)
```

Comment:
Presumably you need to use the credentials that DESNZ gave you. It's probably obvious, but people might have multiple AWS credentials.

Reply:
Also it would be good to be clear here what AWS infrastructure your local instance will connect to. Presumably it's DEV?